app/controllers/mentors_controller.rb

```diff
@@ -0,0 +1,4 @@
+class MentorsController < ApplicationController
+  def index
+  end
+end
```

Comment:
can we place add ```@page_title = 'Mentors'``` to the index action. this is so we do not need to hard code the page titles and all the pages will look the same.

Reply:
👍